protocol/migrations/sqlite/1658309863_add_discord_message_fields_to_user_messages.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE user_messages ADD COLUMN discord_message_id TEXT DEFAULT "";
+ALTER TABLE user_messages ADD COLUMN discord_message_type VARCHAR DEFAULT "";
+ALTER TABLE user_messages ADD COLUMN discord_message_timestamp INT;
+ALTER TABLE user_messages ADD COLUMN discord_message_timestamp_edited INT;
+ALTER TABLE user_messages ADD COLUMN discord_message_content TEXT DEFAULT "";
+ALTER TABLE user_messages ADD COLUMN discord_message_author_id TEXT DEFAULT "";
```

Comment:
This is analog to how we store other payload times.
Could be worth created an additional table for `discord_messages` and only store a `discord_message_id` here (since not every user_message will have a discord_message).

Lemme know what you think

Reply:
Would moving these columns to a separate table and doing a left join against such table result in a slow query to retrieve messages?

The reason I ask is because it seems to me that adding new columns to `user_messages` table for each messaging platform we want to support doesn't look elegant. If in the future we wanted to support importing messages from matrix / slack / teams / etc (just to give an example), that would mean adding 6 new columns for each platform

Reply:
> Would moving these columns to a separate table and doing a left join against such table result in a slow query to retrieve messages?

Something to measure, but definitely would prefer that!

Reply:
This is now done in https://github.com/status-im/status-go/pull/2784